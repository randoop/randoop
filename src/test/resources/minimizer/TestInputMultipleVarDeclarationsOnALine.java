import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputMultipleVarDeclarationsOnALine {

  // Test input with multiple variable declarations on a single line. The minimizer
  // will not attempt to simplify these statements.
  @Test
  public void test1() throws Throwable {
    // Multiple variable declarations on one line.
    int i, j = 2;
    int k;

    // True, j is equal to 2.
    org.junit.Assert.assertTrue(j == 2);

    i = 1;
    k = 3;

    // Assertion fails.
    org.junit.Assert.assertTrue(i + j == k - 1);
  }
}
