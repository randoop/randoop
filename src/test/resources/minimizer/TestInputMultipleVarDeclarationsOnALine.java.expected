import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputMultipleVarDeclarationsOnALine {
  @Test
  public void test1() throws Throwable {
    // Multiple variable declarations on one line.
    int i, j, k;
    i = 1;
    j = 2;
    k = 3;

    double x = 1.0;
    double y;
    double z;
    y = 2.0;
    z = 3.0;

    float a, b = 2.0;
    float c = 3.0;
    a = 1.0;

    org.junit.Assert.assertTrue(i == x);
    org.junit.Assert.assertTrue(a == x);

    org.junit.Assert.assertTrue(j == y);
    org.junit.Assert.assertTrue(b == y);

    org.junit.Assert.assertTrue(k == z);
    org.junit.Assert.assertTrue(c == z);
  }
}
